test/language/templates/template-values/line-terminator-sequence.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 11.8.6
+description: >
+    The TRV of LineTerminatorSequence :: <LF> is the code unit value 0x000A.
+    The TRV of LineTerminatorSequence :: <CR> is the code unit value 0x000A.
+    The TRV of LineTerminatorSequence :: <CR><LF> is the sequence consisting of
+    the code unit value 0x000A.
```

Comment:
Ah, now I understand. (I think GitHub was initially having some trouble with the `<` and `>` characters in your comment.) This batch of tests is ported from V8 directly, but I can certainly include more assertions. I'll just introduce them with a standalone commit to avoid confusion.